### PR TITLE
feat(assistants_web): new citations style

### PR DESCRIPTION
**Pull request by tomtobac**
## Description

Adds new style for citations

- Remove the citation highlighting global state
- Remove the citation onhover global state

### Screenshots

image

image

image

image

image

### CODE REVIEW

File: src/interfaces/assistants_web/src/app/(main)/(chat)/Chat.tsx [hunk 1]
```diff
@@ -56,7 +56,7 @@ const Chat: React.FC<{ agentId?: string; conversationId?: string }> = ({
 
   // Reset citations and file params when switching between conversations
   useEffect(() => {
-    resetCitations();
+    // resetCitations();
```
Comment (knajjars) [inline src/interfaces/assistants_web/src/app/(main)/(chat)/Chat.tsx:59]: remove?

File: src/interfaces/assistants_web/src/components/Agents/AgentRightPanel.tsx [hunk 2]
```diff
@@ -220,7 +215,7 @@ const AgentRightPanel: React.FC<Props> = () => {
           </Text>
         </section>
       </div>
-      <CitationsTab />
+      <></>
```
Comment (knajjars) [inline src/interfaces/assistants_web/src/components/Agents/AgentRightPanel.tsx:218]: remove

File: src/interfaces/assistants_web/src/components/Shared/Icon/Icon.tsx [hunk 3]
```diff
@@ -976,14 +976,14 @@ const getIcon = (name: IconName, kind: IconKind) => {
...
-            clip-rule="evenodd"
+            fillRule="evenodd"
+            clipRule="evenodd"
             d="M3.2 12.9996C3.2 12.9996 3 12.9996 3 12.7996V7.39961C3 7.19961 2.9 7.09961 2.7 7.09961H2.3C2.1 7.09961 2 7.19961 2 7.39961V13.5996C2 13.7996 2.2 13.9996 2.4 13.9996H8.7C8.9 13.9996 8.9 13.9996 8.9 13.6996V13.1996C8.9 12.9996 8.9 12.9996 8.6 12.9996H3.2Z"
             fill="inherit"
           />
           <path
-            fill-rule="evenodd"
-            clip-rule="evenodd"
+            fillRule="evenodd"
```
Comment (knajjars) [inline src/interfaces/assistants_web/src/components/Shared/Icon/Icon.tsx:985]: thanks lol

File: src/interfaces/assistants_web/src/constants.ts [hunk 4]
```diff
@@ -26,20 +26,7 @@ export const ACCEPTED_FILE_TYPES: FileAccept[] = [
 ];
 export const MAX_NUM_FILES_PER_UPLOAD_BATCH = 50;
 
-/**
- * Special HTML ids and CSS classes
- */
-// Classes used to identify certain elements
-export enum ReservedClasses {
```
Comment (knajjars) [inline src/interfaces/assistants_web/src/constants.ts:33]: feels good

**Review by knajjars — APPROVED** [review]
image

File: src/interfaces/assistants_web/src/app/(main)/(chat)/Chat.tsx (hunk 1 above)
Comment (tomtobac) [reply src/interfaces/assistants_web/src/app/(main)/(chat)/Chat.tsx:59]: discussed and agreed to removed

**Pull request merged**